api/src/pcapi/core/offerers/exceptions.py

```diff
@@ -69,6 +69,10 @@ class OffererAddressLabelAlreadyUsed(Exception):
     pass
 
 
+class OfferAddressNotEditableException(Exception):
```

Comment:
`Offer` ou `Offerer` ? Intuitivement je devinerais plutôt : 
```suggestion
class OffererAddressNotEditableException(Exception):
```